v2/Makefile

```diff
@@ -0,0 +1,31 @@
+VERSION     ?= 2.0.0-rc1
+GIT_USER_ID ?= storageos
+GIT_REPO_ID ?= go-api
+
+.DEFAULT_GOAL:=help
+
+.PHONY: help customise
+
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n\nTargets:\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[36m%-13s\033[0m %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
+
+build: generate customise ## Re-generates the API from openapi.yaml
+
+generate:
```

Comment:
`##` comment for this to appear in the make list of commands.

Reply:
done